DxDispatch/src/dxdispatch/DmlDispatchable.h

```diff
@@ -16,11 +34,32 @@ class DmlDispatchable : public Dispatchable
 private:
     std::string m_name;
     std::shared_ptr<Device> m_device;
-    const Model::DmlDispatchableDesc& m_desc;
-    Dispatchable::Bindings m_initBindings;
+    std::variant<Model::DmlDispatchableDesc, Model::DmlSerializedGraphDispatchableDesc> m_desc;
+    Dispatchable::Bindings m_dmlInitBindings;
+    Dispatchable::Bindings m_serializedGraphBindings;
+    bool m_isSerializedGraph = false;
     Microsoft::WRL::ComPtr<IDMLOperator> m_operator;
-    Microsoft::WRL::ComPtr<IDMLCompiledOperator> m_operatorCompiled;
+    Microsoft::WRL::ComPtr<IDMLCompiledOperator> m_compiledOperator;
     Microsoft::WRL::ComPtr<ID3D12Resource> m_persistentBuffer;
     Microsoft::WRL::ComPtr<IDMLBindingTable> m_bindingTable;
     Microsoft::WRL::ComPtr<ID3D12DescriptorHeap> m_descriptorHeap;
+
+    BindPoints m_bindPoints;
```

Comment:
We generally make a method class level only if that method is dependent on any class level fields. All free standalone helper method should be kept inside the cpp file. So some of the below methods declaration should be moved out from here.

Reply:
done